frontend/server/src/main/java/com/amazonaws/ml/mms/wlm/WorkerThread.java

```diff
@@ -110,9 +113,23 @@ public void run() {
                     throw new WorkerInitializationException(
                             "Backend worker did not respond in given time");
                 }
+                switch (req.getCommand()) {
+                    case PREDICT:
+                        model.resetFailedInfReqs();
+                        break;
 
-                if (req.getCommand().equals(WorkerCommands.PREDICT)) {
-                    model.resetFailedInfReqs();
+                    case LOAD:
+                        if ("200".equals(reply.getCode())) {
+                            listener.notifyChangeState(WorkerStateListener.WORKER_MODEL_LOADED);
+                        } else {
+                            listener.notifyChangeState(WorkerStateListener.WORKER_ERROR);
+                        }
+                        break;
+
+                    case UNLOAD:
```

Comment:
We don't use Unload and Stats yet. Why add them here?

Reply:
It is a suggestion from the ide to have them anyway